addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -225,16 +224,16 @@
                                         t-esc="tag_group.name"/>
                                     <div class="dropdown-menu" t-att-id="'navToogleTagGroup%s' % tag_group.id">
                                         <t t-foreach="tag_group.tag_ids" t-as="tag">
-                                            <a rel="nofollow" t-att-class="'dropdown-item %s' % ('active' if tag in search_tags else '')"
-                                                t-att-href="'/slides/all?%s' % keep_query('*', tags=str((search_tags - tag).ids if tag in search_tags else (tag | search_tags).ids))"
+                                            <a rel="nofollow" t-att-class="'dropdown-item post_link %s' % ('active' if tag in search_tags else '')"
```

Comment:
What is "post-link" ?

Reply:
Again, related to multi tags in `GET` request. It's a class used to `POST` the request from JS (so it's not considered as `GET` request and does not remove selected tags).